x-pack/plugins/ml/common/types/notifications.ts

```diff
@@ -5,10 +5,13 @@
  * 2.0.
  */
 
-import type { MessageLevel } from '../constants/message_levels';
+import { ML_NOTIFICATIONS_MESSAGE_LEVEL } from '../constants/notifications';
+
+export type MlNotificationMessageLevel =
+  typeof ML_NOTIFICATIONS_MESSAGE_LEVEL[keyof typeof ML_NOTIFICATIONS_MESSAGE_LEVEL];
```

Comment:
Nit: Suggest to move `MlNotificationMessageLevel` to `'../constants/notifications'` and export from there.